nb/projects/mold/hex.md

```diff
@@ -13,10 +13,7 @@ const assembled = Hexagon(50, 50, 50)
 ![Image](hex.md.0.png)
 
 ```JavaScript
-const disassembled = assembled
-  .each(to(XY()))
-  .pack()
-  .view();
+const disassembled = assembled.each(to(XY())).pack().view();
```

Comment:
What does .each(to(XY())).pack() do? Does't .pack() already do the .each(to(XY)?

Reply:
I didn't change pack yet, probably what we want is something a little more idiomatic to replace .each(to(XY())).